pyqtorch/analog.py

```diff
@@ -640,12 +644,20 @@ def tensor(
         Raises:
             NotImplementedError for the diagonal case.
         """
+
+        values_cache_key = str(OrderedDict(values))
```

Comment:
Wouldn't `OrderedDict(values).keys()` work ?

Reply:
I think not as I save the operator for the values of any parameters and we can obtain a different operator with different inputs.